build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.5
+
+MAINTAINER Peter Mattis <peter@cockroachlabs.com>
+
+RUN apt-get update -y && \
+ apt-get dist-upgrade -y && \
+ apt-get install --no-install-recommends --auto-remove -y git build-essential file npm nodejs && \
+ apt-get clean autoclean && \
+ apt-get autoremove -y && \
+ rm -rf /tmp/* && \
+ ln -s /usr/bin/nodejs /usr/bin/node
+RUN go get golang.org/x/tools/cmd/vet
+
+COPY npm-shrinkwrap.json package.json ./
+RUN npm install
+RUN rm npm-shrinkwrap.json package.json
```

Comment:
Yeah, I combined it with all but the apt-get `RUN`; I wouldn't want `npm` changes to invalidate that cache.